apps/src/lib/cli/client.rs

```diff
@@ -41,8 +41,10 @@ impl<IO> CliApi<IO> {
                         let dry_run = args.tx.dry_run;
                         tx::submit_custom(&client, &mut ctx, args).await?;
                         if !dry_run {
```

Comment:
let's move this expr into `tx::submit_custom` in `apps/src/lib/client/tx.rs` since we already have the `chain_ctx` there